integration/apps/rails-five/README.md

```diff
@@ -0,0 +1,80 @@
+# Rails 5: Demo application for Datadog APM
+
+A generic Rails 5 web application with some common use scenarios.
+
+For generating Datadog APM traces and profiles.
+
+## Installation
+
+Install [direnv](https://github.com/direnv/direnv) for applying local settings.
+
+1. `cp .envrc.sample .envrc` and add your Datadog API key.
+2. `direnv allow` to load the env var.
+3. `cp docker-compose.yml.sample docker-compose.yml` and configure if necessary.
+4. `docker-compose run --rm api bin/setup`
+
+## Running the application
+
+### To monitor performance of Docker containers with Datadog
```

Comment:
I think in this case I lean towards the copy-pasta, because these docs won't change often, and when they do, they may have steps specific to that app. Plus I think there's something to be said for reading everything in one document rather than two.